utils/getRoutes.js

```diff
@@ -3,7 +3,7 @@ export default async () => {
   
   const {
     data: { data },
-  } = await axios.get(`http://localhost:1337/api/blogs?populate=*`);
+  } = await axios.get(`${process.env.strapiUrl}api/blogs?populate=*`);
```

Comment:
We may need that `/` after the base strapi url?

As in the config, we don't have `/`  for `localhost`...

![image](https://github.com/formester/formester-site/assets/69912046/39969a52-62f0-49f0-bae2-05375b573ef4)

Reply:
Yes..